db/sql/05_msar.sql

```diff
@@ -5681,3 +5681,135 @@ Returns:
   FROM tab_info_cte AS tic
   LEFT JOIN col_info_cte AS cic ON tic.tab_id = cic.tab_id
 $$ LANGUAGE SQL RETURNS NULL ON NULL INPUT;
+
+
+CREATE OR REPLACE FUNCTION msar.insert_lookup_table(field_info_list jsonb, values_ jsonb) RETURNS TABLE
```

Comment:
This should be named `build_insert_lookup_table` or `lookup_table_for_insert`.  the current name is super confusing.

Reply:
renamed to `build_insert_lookup_table` in fac1467.